src/NUnitFramework/framework/Internal/Execution/ParallelWorkItemDispatcher.cs

```diff
@@ -82,8 +82,8 @@ public ParallelWorkItemDispatcher(int levelOfParallelism)
 
         private void OnStartNonParallelWorkItem(TestWorker worker, WorkItem work)
         {
-            if (work is CompositeWorkItem)
-                SaveQueueState(work);
+            if (work is CompositeWorkItem && work.Test.TypeInfo != null)
+                IsolateQueues(work);
```

Comment:
I know that `TypeInfo` is only not null when it is a fixture, but that isn't obvious reading this code. At a minimum we need a comment, but might it be better to add a property to `Workitem` or `ITest` like `IsFixture` that wraps this test to make it more explicit? I am just thinking that I shouldn't need to read the code and understand `WorkItem` and it's derived classes to understand this if statement.